src/less/components/javascript/confirmation.less

```diff
@@ -75,8 +75,8 @@ This components data-api only supports `<button>` elements.
 </table>
 
 <h5>Methods</h5>
-<h6>$().confirm(options)</h6>
-Creates a click on the element that triggers a confirmation dialog.
+<h6>$().showConfirmation()</h6>
+Triggers a confirmation dialog on the given element.
```

Comment:
I think not. Can we merge anyway and then ask someone with more English language skills?